src/control/server/ioserver/config.go

```diff
@@ -186,6 +187,20 @@ func (c *Config) CmdLineEnv() ([]string, error) {
 		return nil, err
 	}
 
+	// Provide special handling for the ofi+verbs provider.
+	// When specified, perform a special lookup to find the alias
+	// of the given fabric_iface.  Then set a new environment variable "OFI_DOMAIN"
+	// to the name of the device alias.  For example, a fabric_iface of "ib0"
+	// has a device alias of "hfi1_0", and "OFI_DOMAIN=hfi1_0" will be added
+	// to this I/O server's environment
+	if c.Fabric.Provider == "ofi+verbs" {
```

Comment:
We also sometimes use "ofi+verbs;ofi_rxm" for verbs, will this still work?
We should match "ofi+verbs*".

Reply:
I will update it to address that.

Reply:
I just pushed a version that addresses the provider match.  I am still working on resolving the unit test failure.